src/resolvers/InstallableAppResourceResolver.ts

```diff
@@ -18,11 +20,20 @@ class InstallableAppResourceResolver implements AppResourceResolver, BuiltinReso
     public resolveResource(_subContext: ISubscriptionContext, resource: AppResource): ResolvedAppResourceBase {
         // We know the extension is known but uninstalled, or else it would not have passed the `isApplicable` check below
         // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
-        const extensionId = getAzureExtensions().find(azExt => azExt.matchesResourceType(resource))!.id;
+        const extension = getAzureExtensions().find(azExt => azExt.matchesResourceType(resource))!;
 
         return {
-            commandId: 'extension.open',
-            commandArgs: [extensionId],
+            loadMoreChildrenImpl: async () => {
+                const ti = new GenericTreeItem(undefined, {
+                    contextValue: 'installExtension',
+                    label: localize('installExtension', 'Install {0} extension...', extension.label),

```

Comment:
It might be too cluttered, but it would be good if we could include some text explaining why a user would want to install the extension.

Maybe something like:
`Install {0} extension for enhanced features...` or something like that.

Reply:
We need Matt for this... 🪄✨

Reply:
Updated to this:
<img width="378" alt="image" src="https://user-images.githubusercontent.com/12476526/160712292-d949aa0d-347e-4357-aea8-802ff48c2cae.png">